vexim/admingroupchange.php

```diff
@@ -106,7 +106,7 @@ class="textfield">
                   </a>
                 </td>
                 <td><?php echo $row['realname']; ?></td>
-                <td><?php echo $row['username']; ?></td>
+                <td><?php echo $row['localpart'].'@'.$_SESSION['domain']; ?></td>
```

Comment:
I wonder if joining the query with the `domains` table and using the domain name from that table wouldn't be a better idea.